src/app/shared/components/delegated-acc-message/delegated-acc-message.component.ts

```
@@ -1,20 +1,39 @@
   templateUrl: './delegated-acc-message.component.html',
   styleUrls: ['./delegated-acc-message.component.scss'],
 })
 export class DelegatedAccMessageComponent implements OnInit {
-  delegateeName;
+  delegateeName: string;
+  isConnected$: any;
 
-  constructor(private authService: AuthService) {}
+  constructor(private authService: AuthService, private router: Router, private networkService: NetworkService) {}
 
   ngOnInit() {
```

ESLint

[error] Missing return type on function.

(@typescript-eslint/explicit-function-return-type)